### PR TITLE
research highlight refresh

### PR DESCRIPTION
**Pull request by freeman-lab**
Still todo:
- [x] see if the same (or similar, or lightly modified) graphics work on the `/research` highlight page
- [x] replace links to images hosted on our CDN (rather than versions hosted on `/public`)

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -34,6 +34,8 @@ const sx = {
 }
 
 const Index = () => {
+  const [colorMode] = useColorMode()
```
Comment (katamartin) [inline pages/index.js:37]: It doesn't seem like this is being used in this component -- can we remove?

File: components/highlight.js [hunk 2]
```diff
@@ -57,6 +59,28 @@ const Highlight = ({ research }) => {
           </Box>
         </Column>
       </Row>
+      <Row as='figure' sx={{ mt: [4, 5, 6, 7], mb: [4, 4, 4, 5] }}>
+        <Column start={[1, 1, 2, 3]} width={[6, 8, 10, 8]}>
+          <Link href={research.href}>
+            <Image
```
Comment (katamartin) [inline components/highlight.js:65]: Perhaps depending on how complicated the height-setting logic becomes (e.g. maybe less necessary if there's a simple solution using `aspect-ratio`), we might consider formalizing this `Image` pattern by building into `components`.

**Review by katamartin — COMMENTED** [review]
left a few comments, but largely lgtm!

File: components/highlight.js (hunk 2 above)
Comment (freeman-lab) [reply components/highlight.js:69]: yeah maybe we wait for at least a third use case? there's a lot here that's pretty specific to these images (and their aspect ratios) and these layouts (although the two cases have a lot in common, not clear yet whether that will generalize)

**Pull request merged**